guide/lib/helpers/link_helpers.rb

```diff
@@ -24,6 +24,7 @@ def navigation_links
           "Checkboxes" => "/form-elements/checkboxes/",
           "Date input" => "/form-elements/date-input/",
           "File upload" => "/form-elements/file-upload/",
+          "Password fields" => "/form-elements/password/",
           "Radios" => "/form-elements/radios/",
```

Comment:
As previous comment, and also to be internally consistent with URL slugs:

```suggestion
          "File upload" => "/form-elements/file-upload/",
          "Password input" => "/form-elements/password-input/",
          "Radios" => "/form-elements/radios/",
```